tidy3d/plugins/adjoint/components/data/data_array.py

```diff
@@ -312,30 +313,76 @@ def isel(self, **isel_kwargs) -> JaxDataArray:
 
         return self_sel
 
-    def sel(self, indexers: dict = None, method: str = "nearest", **sel_kwargs) -> JaxDataArray:
-        """Select a value from the :class:`.JaxDataArray` by indexing into coordinate values."""
+    def sel(
+        self, indexers: dict = None, method: Literal[None, "nearest"] = None, **sel_kwargs
+    ) -> JaxDataArray:
+        """Select a value from the :class:`.JaxDataArray` by indexing into coordinates by value.
+
+        Parameters
+        ----------
+        method : Literal[None, "nearest"] = None
```

Comment:
we'll want to add `sel_kwargs` I think here (probably first parameter, just for the docs display). Just so users understand that they need to pass key: value pairs corresponding to the keys and values of the `.coords` they want to select.

Reply:
added

Reply:
sorry, nitpick-y comment, but I think saying that it's a dictionary might be confusing. Could we just change "Dictionary" to "Keyword arguments"?